code/style.css

```diff
@@ -1,7 +1,268 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  background:rgb(211, 226, 251);
+  color: navy;
+}
+
+/********** HERO-IMAGE  ************/
+.hero {
+  position: relative;
+}
+
+.hero-image {
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+}
+
+.hero-title {
+  position: absolute;
+  top:0;
+  left: 50px;
+  font-size: 60px;
+  color: rgb(212, 11, 212);
+}
+
+/****************** SIGN UP FORM  ************************/
+input, textarea, button {
+  border-radius: 5px;
+}
+
+.itext {
+  box-shadow: 2px 3px 2px 2px rgba(7, 74, 100, 0.4);
+  height: 22px;
+  padding: 2px;
+}
+
+textarea {
+  box-shadow: 2px 3px 2px 2px rgba(7, 74, 100, 0.4);
+  background:rgb(220, 223, 245)
+}
```

Comment:
`resize: none;` is a great property to add for text areas as well so that the user is prevented from resizing and messing up the styling ;)